src/IECoreGL/TextureLoader.cpp

```diff
@@ -37,19 +37,27 @@
 #include "IECoreGL/Texture.h"
 #include "IECoreGL/ToGLTextureConverter.h"
 
+#include "IECoreImage/ImageReader.h"
```

Comment:
Is this dependency acceptable?

Reply:
Yep. IECoreGL has a dependency on IECoreImage already.